pkg/cmd/root.go

```diff
@@ -125,12 +128,9 @@ func Execute(ctx context.Context) {
 		}
 
 		os.Exit(1)
-	} else {
```

Comment:
i think moving the `userInput` (so i could reuse it) created a lint error for this; since it's just a minor structural change i think it's ok?